opencog/atoms/pattern/BindLink.h

```diff
@@ -52,7 +52,8 @@ class BindLink : public PatternLink
 	BindLink(const Handle& body, const Handle& rewrite);
 	explicit BindLink(const Link &l);
 
-	bool imply(PatternMatchCallback&, bool check_connectivity=true);
+	bool imply(PatternMatchCallback&, AtomSpace* as,
+	           bool check_connectivity=false);
 	const Handle& get_implicand(void) { return _implicand; }
```

Comment:
why change the default connectivity check?  The connectivity check is intended to avoid combinatorial explosions - its a performance issue.

Reply:
It was already set to false, `bindlink` the main entry point for using the pattern matcher sets it to false https://github.com/opencog/atomspace/blob/master/opencog/query/Implicator.cc#L90

the only places it wasn't set to false were a couple unit tests (failing as a result of these changes). If you prefer we can revert that and set to false only for the couple of failing unit tests, we just thought it makes more sense to set this one to false since again, for all practical purpose, it was already set to false.